test/e2e/atlas/clusters_upgrade_test.go

```diff
@@ -102,35 +65,27 @@ func TestSharedClusterUpgradeToDedicatedTier(t *testing.T) {
 			"-o=json")
 		cmd.Env = os.Environ()
 		resp, err := cmd.CombinedOutput()
-		req.NoError(err, string(resp))
+		require.NoError(t, err, string(resp))
 	})
+	require.NoError(t, watchCluster(g.projectID, g.clusterName))
+	ensureClusterTier(t, cliPath, g.projectID, g.clusterName, tierM10)
```

Comment:
as this upgrade also changes the disk size, mdbVersion, and the tags, can we validate all these changes, not only the tier? (unless covered by another test)

Reply:
the version change is a lie, the current default on shared tiers, probably at the time of writing this test the default of shared was 5.0 but it changes with time. I removed that flag from the update and added back the checks for disk size and implemented a check for tags

Test are still running so we'll wait for the test to pass